### PR TITLE
datatools.SOWFA6 module

### PR DESCRIPTION
**Pull request by DriesAllaerts**
Adding a separate `datatools.SOWFA6` module with SOWFA6 specific data loaders:
- `averaging.py`
- `probes.py`
- `sourceHistory.py`

The new module only contains SOWFA6 specific classes and functions, i.e., scripts like `SOWFA/postProcessing/sliceDataInstantaneous.py` or `SOWFA/constant` are not copied over in the new SOFWA6 module.

The original `SOWFA/postProcessing` module has been cleaned up, removing all new data loaders actually corresponding to SOWFA6 format (e.g., `averaging.PlanarAvergeFunctionObject`)

The SOWFA6 postProcessing classes have been implemented using a unified reader class (see previous pull request #10 ):
_The classes in `averaging.py`, `probes.py` and `sourceHistory.py` share a lot of code: determining the number of time directories, reading all time directories and appending data, removing overlapped data due to restarts, trimming time series, representation, and output functions to pandas, csv and netcdf. This pull request combines these parts in a new reader class, and the other classes inherit from this general reader class. What is left in the specialised classes is simple readers for the specific file formats._

### CODE REVIEW

File: SOWFA/postProcessing/averaging.py [hunk 1]
```diff
@@ -1156,293 +1156,6 @@ def to_netcdf(self,fname):
 
 """end of class PlanarAverages"""
 
-class PlanarAverageFunctionObject(object):
```
Comment (ewquon) [inline SOWFA/postProcessing/averaging.py:1159]: The function object is only going to be implemented within SOWFA6, right?

File: SOWFA6/postProcessing/averaging.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+from matplotlib import cm
+from datatools.SOWFA6.postProcessing.reader import Reader
+
+class PlanarAverages(Reader):
+
+    def __init__(self,dpath=None,**kwargs):
+        super().__init__(dpath,includeDt=True,**kwargs)
+
+
+    def _processdirs(self,
```
Comment (ewquon) [inline SOWFA6/postProcessing/averaging.py:40]: Can all of these `_processdirs`, `_read_*` be moved into the `Reader` class?

File: SOWFA/postProcessing/averaging.py (hunk 1 above)
Comment (DriesAllaerts) [reply SOWFA/postProcessing/averaging.py:1159]: That is correct

File: SOWFA6/postProcessing/averaging.py (hunk 2 above)
Comment (ewquon) [reply SOWFA6/postProcessing/averaging.py:40]: Nevermind, I understand the overridden functions now.

**Pull request merged**